libs/watt/package/dropzone/watt-dropzone.ts

```diff
@@ -0,0 +1,175 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import {
+  Component,
+  booleanAttribute,
+  forwardRef,
+  inject,
+  input,
+  output,
+  signal,
+} from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR, ReactiveFormsModule } from '@angular/forms';
+import { WattButtonComponent } from '@energinet/watt/button';
+import { WattFieldComponent } from '@energinet/watt/field';
+import { VaterStackComponent, VaterUtilityDirective } from '@energinet/watt/vater';
+import { WattDropZoneIntlService } from './watt-dropzone-intl';
+
+// Slightly better typing than just raw string
+type MimeType = `${string}/${string}`;
```

Comment:
Suggestion: Maybe this could be used https://www.npmjs.com/package/@types/mime-types ?

Reply:
If you look at the `index.d.ts` file of that package (https://www.npmjs.com/package/@types/mime-types?activeTab=code), it just looks to be types for the `mime-types` package. We could of course use the `mime-types` package to map extensions to MIME types during runtime (so that we can use extensions for accept input), but that is super not worth it, since the package pulls in `mime-db`, which is a 200KB+ JSON file.

Perhaps you were thinking something more like this (https://github.com/samhuk/mime-types-typescript/blob/master/dist/latestUpdate.ts)? That is not a package though and I'm not sure it is worth it either, since that list has to be maintained as new MIME types are added.

Reply:
I just wanted it for type completion when coding, should not be a part of the runtime

Reply:
I'm all for type completion, but I think in this case the benefit from the type completion does not outweight the cost of maintenance. I could only find this repo (https://github.com/samhuk/mime-types-typescript), and that requires us to manually rebuild the list occasionally (I don't know how often that is though).